KuringApp/KuringApp/Subscription/SubscriptionView.swift

```diff
@@ -51,74 +81,205 @@ struct SubscriptionFeature: Reducer {
+                                }
+                                
+                            }
+                        }
+                        Spacer()
+                    }
+                case .department:
+                        if viewStore.myDepartments.isEmpty {
+                            VStack(alignment: .center) {
+                                Spacer()
+                                HStack {
+                                    Spacer()
+                                    Text("아직 추가된 학과가 없어요.\n관심 학과를 추가하고 공지를 확인해 보세요!")
+                                        .font(.system(size: 16, weight: .medium))
+                                        .multilineTextAlignment(.center)
+                                        .foregroundColor(Color(red: 0.68, green: 0.69, blue: 0.71))
+                                    Spacer()
+                                }
+                                Spacer()
+                            }
+                        } else {
+                            VStack(spacing: 0) {
+                                ScrollView(showsIndicators: false) {
+                                    ForEach(viewStore.myDepartments) { department in
+                                        let isSelected = viewStore.selectedDepartment.contains(department)
+                                        
+                                        VStack(spacing: 0) {
+                                            HStack {
+                                                Text(department.id)
+                                                    .font(.system(size: 16, weight: .semibold))
+                                                    .foregroundColor(.black)
+                                                Spacer()
+                                                Image(systemName: isSelected ? "checkmark.circle.fill" : "circle")
+                                                    .foregroundStyle(isSelected ? Constants.kuringPrimary : Color.black.opacity(0.1))
+                                                    .frame(width: 20, height: 20)
+                                            }
+                                            .padding(.horizontal, 21.5)
+                                            .padding(.top, viewStore.myDepartments.first?.id == department.id ? 22 : 0)
+                                            .padding(.bottom, viewStore.myDepartments.last?.id == department.id ? 22 : 0)
+                                            .contentShape(Rectangle())
+                                            .onTapGesture {
+                                                viewStore.send(.departmentSelected(department))
+                                            }
+                                            
+                                            // 밑줄
+                                            if viewStore.myDepartments.last?.id != department.id {
```

Comment:
Department 가 equatable 이라면 아래처럼 바꾸는게 어떨까요
```suggestion
                                            if viewStore.myDepartments.last != department {
```

Reply:
위에 first 사용하는 로직도 동일

Reply:
https://github.com/ku-ring/ios-app/pull/32/commits/b5b77dbdab76b5a963c4aeacae5a10bcc118733d